activity_browser/layouts/tabs/activity.py

```diff
@@ -144,6 +155,9 @@ def __init__(self, key: tuple, parent=None, read_only=True):
             ("Downstream Consumers:", self.downstream),
         ]
         self.grouped_tables = [DetailsGroupBox(l, t) for l, t in self.exchange_tables]
+        if ActivityTab.open_table_toggles:
```

Comment:
I'm not entirely sure why this is in an if statement, we always create `open_table_toggles` before this if, so it will always be true. If you want to know it contains something it should be `if len(..) > 0`, though also for that, we haven't put anything inside the list yet so `len` would always be 0?

Reply:
that's odd, what version of Python are you using? For me with version 3.9 the logical evaluation of an empty list comes to False, which was my intention. However thinking about it the for loop won't execute when the length of the list is zero, so this is redundant.

Reply:
Oh you may be right on that actually. I just thought this was the case. Sorry for confusing you there

Reply:
Well your comment made me realise it was redundant, so its good to confuse me!